.github/workflows/tests.yml

```diff
@@ -0,0 +1,62 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  tests:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [7.1, 7.2, 7.3, 7.4]
+        laravel: [5.7.*, 5.8.*, ^6.0, ^7.0]
+        phpunit: [^7.5, ^8.0]
+        exclude:
+          - php: 7.1
+            laravel: 5.7.*
+            phpunit: ^8.0
+          - php: 7.1
+            laravel: 5.8.*
+            phpunit: ^8.0
+          - php: 7.1
+            laravel: ^6.0
+            phpunit: ^7.5
+          - php: 7.1
+            laravel: ^6.0
+            phpunit: ^8.0
+          - php: 7.1
+            laravel: ^7.0
+            phpunit: ^7.5
+          - php: 7.1
+            laravel: ^7.0
+            phpunit: ^8.0
+
+    name: P${{ matrix.php }} - L${{ matrix.laravel }} - U${{ matrix.phpunit }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
```

Comment:
is there a reason you're not using v2 here?

Reply:
Fixed thanks